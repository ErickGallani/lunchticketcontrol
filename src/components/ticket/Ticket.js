import React from 'react';
import PropTypes from 'prop-types';
import './ticket.css';

export default class Ticket extends React.Component {
    constructor(props){
        super(props);
    }
    
    render() {      
      return (
        <div className="ticket">
            <p>Test</p>
        </div>
      );
    }
  }
  
  