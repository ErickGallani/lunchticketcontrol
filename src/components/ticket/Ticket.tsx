import * as React from 'react';
import * as PropTypes from 'prop-types';
import './ticket.css';

export default class Ticket extends React.Component {
    constructor(props: any){
        super(props);
    }
    
    render() {      
      return (
        <div className="ticket">
            <p>Test</p>
        </div>
      );
    }
  }
  
  