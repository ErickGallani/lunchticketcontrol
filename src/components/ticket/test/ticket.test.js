import React, { Component } from 'react';
import Ticket from "../Ticket";
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

test('renders correctly ticket', () => {
  const elem = shallow(<Ticket />);
  expect(shallowToJson(elem)).toMatchSnapshot();
});