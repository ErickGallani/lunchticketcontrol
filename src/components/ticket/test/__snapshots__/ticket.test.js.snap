// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly ticket 1`] = `
<div
  className="ticket"
>
  <p>
    Test
  </p>
</div>
`;
